### DISCUSSION TITLE
Measuring range selection via software

### DISCUSSION THREAD

**Discussion by ludwich66** [webspiderteam/Bluetooth-DMM-For-Windows#30] [Ideas]
Copy Buttons from A-App 162x162 Pixel
[Buttons_162x162.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10023892/Buttons_162x162.zip)

Do you have any idea how to include the function? I hope we can find out the open keys.
I briefly copied the A-App icons here

L

**Comment by webspiderteam** [comment]
Bluetooth connection module don't have sending function i will try to integrate. Then I can add expanding button in main view.

**Comment by ludwich66** [comment]
Verified Switch Codes:
https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/issues/29#issuecomment-1315900948

The software again represents all BT devices. So there are open symbols that the small DMM does not support. Except for "Zero" it should fit for the B05V.

Range °C an °F have its own seperate Code , in App is it represented with double click one button.

ludi incipiant

ludwich

**Comment by webspiderteam** [comment]
@ludwich66 did you see this page before? https://github.com/blackPantherOS/AN9002 there is an app with qt5 and also has wiki page but look like copied from your wik.

**Comment by ludwich66** [comment]
Bilgi için teşekkürler, hayır, o siteyi bilmiyordum. 2 yaşında Her durumda, "güzel bir ipucu" olmadan klonlamanın kibar yolu değildir.
Senin için her şey yolunda mı? (deprem vb. nedeniyle)
Gruß Ludwich

**Comment by webspiderteam** [comment]
also there is apage for android https://gitlab.com/eevblog/app-121gw you need to  take a look.
Its ok here I'm far away from it. thanks for asking.

**Reply by webspiderteam** [reply]
sorry this one is only for bluetooth communucation not for dmm
